web_source/cities/north_america/mexico/mazatlan.md

```diff
@@ -4,12 +4,14 @@ Country: Mexico (N. America)
 
 ##### Also known as:
 
-FIXME
+Mazatlán
 
 ### What we would like to see there
 
-FIXME
+The lighthouse (El Faro) - It has a **1,000 watt bulb**, which is focused by a Fresnel lens!
+
+![a image](https://upload.wikimedia.org/wikipedia/commons/thumb/8/80/Faro_de_Mazatl%C3%A1n.JPG/800px-Faro_de_Mazatl%C3%A1n.JPG)
```

Comment:
It's better to include descriptive image alt-text, not something generic like this!

Reply:
updated the alt text